package.json

```diff
@@ -21,16 +21,16 @@
     "test:coveralls": "nyc npm test"
   },
   "dependencies": {
-    "async": "^2.6.2",
+    "async": "^3.0.1",
```

Comment:
Only thing I'm not sure is the async major version update. It does seem to work fine, though.

Reply:
@jzaefferer: can you have a quick look here please?

Reply:
Seems fine from a long distance away